packages/core/src/config/options-validator.ts

```diff
@@ -97,6 +100,22 @@ export class OptionsValidator {
     }
   }
 
+  private specificMutantsValidation(options: StrykerOptions) {
+    options.mutator.specificMutants = options.mutate
```

Comment:
I can see what you're doing here, but I don't think the parsing should be done here. The validation can move to the `additionalValidation` function, while the parsing should be part of the `InputFileResolver`, where it can provide the range as `mutationRange`.